pallets/cash/src/chains.rs

```diff
@@ -25,13 +25,15 @@ pub enum ChainId {
     Gate,
     Eth,
     Dot,
+    Matic, // xxx todo: due to rebrand ticker and name don't match for MATIC/Polygon.. so review this enum name is this what we want to call this?
```

Comment:
Yeah I think this makes sense, not sure what else we would call the network and this still seems to be what they are frequently calling it